receiver/sqlserverreceiver/scraper_test.go

```diff
@@ -107,8 +106,6 @@ func TestSuccessfulScrape(t *testing.T) {
 	cfg.Port = 1433
 	cfg.Server = "0.0.0.0"
 	cfg.MetricsBuilderConfig.ResourceAttributes.SqlserverInstanceName.Enabled = true
-	cfg.MetricsBuilderConfig.ResourceAttributes.ServerAddress.Enabled = true
```

Comment:
Probably enabling hostname here to keep the previous behavior?

Reply:
updated